client/src/js/directives/bhFormMessage.js

```diff
@@ -0,0 +1,73 @@
+angular.module('bhima.directives')
+.controller('bhFormMessageController', bhFormMessageController)
+.directive('bhFormMessage', bhFormMessage);
```

Comment:
@jniles Had removed this directive from the repository because there are now improved ways to do this. We should have a meeting this morning about how to do this and that should be followed directly by re-factoring and removing this file from master if that is appropriate.